test/acceptance/typography.test.js

```diff
@@ -41,8 +41,8 @@ var browsers = {
 	})
 };
 
-var endpoint = 'http://localhost:8080/components/d2l-typography/test/acceptance/typography.html';
-var endpointMixins = 'http://localhost:8080/components/d2l-typography/test/acceptance/typography-mixins.html';
+var endpoint = 'http://localhost:8000/components/d2l-typography/test/acceptance/typography.html';
+var endpointMixins = 'http://localhost:8000/components/d2l-typography/test/acceptance/typography-mixins.html';
```

Comment:
Not for this PR, but it'd be cool to have this test both the 1.x and 2.x variants.